server/remote/msgraph/remote.go

```diff
@@ -85,3 +86,11 @@ func (r *impl) NewOAuth2Config() *oauth2.Config {
 		Endpoint: microsoft.AzureADEndpoint(r.conf.OAuth2Authority),
 	}
 }
+
+func (r *impl) CheckConfiguration(cfg config.StoredConfig) error {
+	if cfg.OAuth2ClientID == "" || cfg.OAuth2ClientSecret == "" || cfg.OAuth2Authority == "" {
+		return fmt.Errorf("OAuth2 credentials to be set in the config")
```

Comment:
Nice to have: Probably good to name the config settings as they are shown in the plugin config page, e.g. `Azure Directory (tenant) ID`

Reply:
I'd rather do this later on and focus on gcal related things at the moment. We can do that once we change the mscalendar` name to something different and address all in the same PR.